llpc/tool/llpcInputUtils.cpp

```diff
@@ -313,22 +315,36 @@ static void findAllMatchFiles(const std::string &inFile, std::vector<std::string
 // @returns : Result::Success on success, Result::ErrorInvalidValue when expansion fails.
 Result expandInputFilenames(ArrayRef<std::string> inputSpecs, std::vector<std::string> &expandedFilenames) {
   unsigned i = 0;
-  for (const auto &inFile : inputSpecs) {
+  for (const auto &rawInFile : inputSpecs) {
 #ifdef WIN_OS
     {
+      // Handle any optional entry point after the filename (inputSpecs can be
+      // of the form <filename>,<entrypoint> - in this case strip the
+      // <entrypoint> from the inputSpec then pass in as option to findAllMatchFiles
+      size_t found = rawInFile.find_last_of(",?");
+      std::string entryPoint;
+      std::string inFile;
+      if (found != std::string::npos) {
+        entryPoint = rawInFile.substr(found);
+        inFile = rawInFile.substr(0, found);
+      } else {
+        inFile = rawInFile;
+      }
+
       if (i > 0 && inFile.find_last_of("*?") != std::string::npos) {
         LLPC_ERRS("\nCan't use wilecards with multiple inputs files\n");
         return Result::ErrorInvalidValue;
```

Comment:
Can we have a similar check that makes sure wildcards and entryPoints are not provided at the same time. And then drop the entryPoint argument to `findAllFilesMatch`?

Reply:
Yes, that make sense.